sensing/imu_corrector/src/imu_corrector_core.cpp

```diff
@@ -12,7 +12,9 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-#include "imu_corrector/imu_corrector_core.hpp"
+#include "imu_corrector_core.hpp"
+
+#include <algorithm>
```

Comment:
Why did you add <algorithm> in this code?
I tried building and simulation with line 17 removed, but it seems to work fine.

Reply:
Used in std::max, etc
It is true that the compilation of the code without the inclusion in .cpp also works since it's included somewhere else (in this case probably from `imu_corrector_core.hpp`). But then the pre-commit fails:
```cpplint..................................................................Failed
- hook id: cpplint
- exit code: 1

sensing/imu_corrector/src/imu_corrector_core.cpp:24:  Add #include <algorithm> for max  [build/include_what_you_use] [4]
Done processing sensing/imu_corrector/src/imu_corrector_core.cpp
Total errors found: 1
```

(Which often called as "include-what-you-use" practice)